scripts/render-metrics-gif.py

```
@@ -34,163 +46,467 @@
+
+    if attacks:
+        det_by_attack, phase_attacks = build_attack_analysis(attacks, detections)
+
+        total_per_phase = []
+        detected_per_phase = []
+        for p in MITRE_PHASES:
+            pa = phase_attacks[p]
+            total_per_phase.append(len(pa))
+            detected_per_phase.append(sum(1 for a in pa if a["all_found"]))
+
+        max_radar = max(max(total_per_phase), 1)
```

Ruff (0.15.12)

[warning] Nested `max` calls can be flattened

Flatten nested `max` calls

(PLW3301)